book/src/kubernetes-cluster-controllers.md

```diff
@@ -0,0 +1,7 @@
+# Kubernetes Cluster Controllers
+
+The Kubernetes cluster controllers should be bundled and deployed by the Kubernetes distributors as part of their Kubernetes cluster management process (independent of any CSI Driver).
```

Comment:
maybe also mention that the controllers are responsible for managing snapshot objects and operations across multiple CSI drivers, so that's why they should not be deployed by the driver.

Reply:
Done